app/presenters/provider/admin/backend_apis_index_presenter.rb

```diff
@@ -3,39 +3,43 @@
 class Provider::Admin::BackendApisIndexPresenter
   include System::UrlHelpers.system_url_helpers
 
-  def initialize(current_account:, params: {})
-    @current_account = current_account
+  def initialize(current_user:, params: {})
+    @current_user = current_user
     @pagination_params = { page: params[:page] || 1, per_page: params[:per_page] || 20 }
     @sorting_params = "#{params[:sort].presence || 'updated_at'} #{params[:direction].presence || 'desc'}"
     @search = ThreeScale::Search.new(params[:search])
+    @ability = Ability.new(current_user)
   end
 
-  attr_reader :current_account, :pagination_params, :sorting_params, :search
+  attr_reader :current_user, :pagination_params, :sorting_params, :search, :ability
```

Comment:
I don't think `@current_user` and `@ability` should be publicly accessible unless that's really needed. Same for all other occurrences.